picasso/src/main/java/com/squareup/picasso/PicassoDrawable.java

```diff
@@ -85,63 +81,22 @@ static void setPlaceholder(ImageView target, Drawable placeholderDrawable) {
 
     boolean fade = loadedFrom != MEMORY && !noFade;
     if (fade) {
-      this.placeholder = placeholder;
-      animating = true;
-      startTimeMillis = SystemClock.uptimeMillis();
+      startTransition(FADE_DURATION);
+    }
+    else {
+      // Transition straight to the final drawable
```

Comment:
suuuuupernit: period at the end. [Comments should be sentences](http://nedbatchelder.com/blog/201401/comments_should_be_sentences.html).